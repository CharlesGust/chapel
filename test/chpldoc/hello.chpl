/**
 * This method prints out a greeting.
 */
proc hello() {
  writeln("Hello, world!");
}