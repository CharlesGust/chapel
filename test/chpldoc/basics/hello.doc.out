Module: hello
   hello()
       This method prints out a greeting.

